config/config-builder/src/key_manager_config.rs

```diff
@@ -15,6 +15,7 @@ pub struct KeyManagerConfig {
 
     pub vault_host: String,
     pub vault_namespace: Option<String>,
+    pub vault_network_timeout_ms: u64,
```

Comment:
seems that this either needs to be optional or defined everywhere and one of the config doesn't have it defined... probably best to make this optional and then an unwrap_or(DEFAULT_VAULT_NETWORK_TIMEOUT_MS)